src/main/kotlin/inc/evil/d17map/dtos/ReservationRequest.kt

```diff
@@ -34,7 +35,12 @@ data class ReservationRequest(
     val type: ReservationType,
 
     @field:Positive(message = "Number of participants must be at least 1.")
-    val numberOfParticipants: Int
+    val numberOfParticipants: Int,
+
+    var recurringId: UUID? = null,
```

Comment:
I'm wondering if that shouldn't be added by backend? It has to be unique and on the frontend side we don't know what is a unique value for recurringId

Reply:
this argument is optional, you don't have to provide it